### ISSUE TITLE
<input type="file"> should use 'multiple' and 'accepts' attributes

### ISSUE THREAD

**Issue by nickzoic** [Andereoo/TkinterWeb#68]
See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file

Multiple seems pretty simple: use askopenfilenames, and (like firefox) just display "2 files selected" instead of the filename if there's 2 of them, etc.

Accepts is maybe trickier as it is a comma separated list of extensions and/or mime types but the askopenfilename(s) methods expect an array of (label,glob) pairs ... presumably the browser translates.

**Comment by Andereoo** [comment]
Thanks for the suggestion! I implemented the features you suggested.
Upgrade TkinterWeb, and let me know what you think!

**Comment by nickzoic** [comment]
Looks good!  I was going to suggest using `mimetypes` library for the mime type translation ... `mimetypes.guess_all_extensions("image/*")` doesn't work, sadly, but we could do something like this to handle the special case:

```
def mimetype_to_extensions(mimetype):
    if mimetype.endswith("*"):
        return [ k for k, v in mimetypes.types_map.items() if v.startswith(mimetype[:-1]) ]
    else:
       return mimetypes.guess_all_extensions(mimetype)
```
I can throw together a PR if you like.

**Comment by Andereoo** [comment]
Great idea! I completely forgot `mimetypes` existed...

If you felt like throwing together a PR that would be great. Otherwise, I'll figure something out in the next few days.

**Comment by nickzoic** [comment]
I've also added in some code to give a "All Supported Types" default option which is much more like what Firefox does.

Actually I just looked up the standard https://html.spec.whatwg.org/multipage/input.html#file-upload-state-(type=file) and I've over-reached a bit with matching any MIME type ending in `*` ... officially only `audio/*`, `video/*` and `image/*` are allowed.  So we can roll that back a little if you like by replacing the `mimetype.endswith("*")` clause.

Also the spec notes "_Authors are encouraged to specify both any MIME types and any corresponding extensions when looking for data in a specific format._" which means we'll end up with a lot of repetition in the list.  It'd be nice to detect this case, so if you have `accepts=".jpg,.jpeg,image/jpeg"` it can expand `image/jpeg` to `('image/jpeg', '.jpg .jpe .jpeg .jfif')`, and notice that the other cases are already covered so just leave them out.
